Sources/Basics/FileSystem+Extensions.swift

```diff
@@ -17,13 +20,93 @@ extension FileSystem {
     }
 }
 
+
+// MARK: - cache
+
 extension FileSystem {
+    private var idiomaticUserCacheDirectory: AbsolutePath? {
+        // in TSC: FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask)
+        self.cachesDirectory
```

Comment:
Is this ever `nil`?

Reply:
it could be if there is no idiomatic location, iow the Fondation API can return nil